packages/validator/src/validator.ts

```diff
@@ -121,6 +121,10 @@ export class Validator {
     }
 
     const emitter = new ValidatorEventEmitter();
+    // Validator event emitter can have more than 10 listeners in a normal course of operation
+    // We set infinity to prevent MaxListenersExceededWarning which get logged when listeners > 10
+    emitter.setMaxListeners(Infinity);
```

Comment:
Can you add proof with stack trace that this event listener triggers warnings?

Reply:
Already responded to the diff in the comment [here](https://github.com/ChainSafe/lodestar/pull/4421#discussion_r953578392)